subsys/metrics/src/main/java/org/commonjava/indy/metrics/zabbix/sender/IndyZabbixSender.java

```diff
@@ -25,52 +18,52 @@
 import java.util.regex.Pattern;
 
 /**
- * Created by xiabai on 4/11/17.
+ * Created by xiabai on 4/1/17.
  */
 public class IndyZabbixSender
 {
     private static final Logger logger = LoggerFactory.getLogger( IndyZabbixSender.class );
 
-    private boolean bCreateNotExistHostGroup = true;
+    ZabbixSender sender;
```

Comment:
Do we need these variables to be a broader scope than `private`? In fact, can they be `private final`?